src/eva/vision/data/datasets/segmentation/consep.py

```diff
@@ -37,8 +37,8 @@ def __init__(
         root: str,
         sampler: samplers.Sampler | None = None,
         split: Literal["train", "val"] | None = None,
-        width: int = 224,
-        height: int = 224,
+        width: int = 250,
+        height: int = 250,
```

Comment:
lets do 256 or 224

Reply:
250 is intentional here because the original image is 1000x1000, and we want to divide it into 4x4 patches.
Resisizing to 224 is done via a transformation afterwards.